dotcom-rendering/src/web/components/SignInGate/gateDesigns/shared.tsx

```diff
@@ -28,6 +29,15 @@ export const headingStyles = css`
 	}
 `;
 
+export const personalisedHeadingStyles = css`
```

Comment:
I would move move all of these personalised and bullet styles out of the `shared` file and colocate the css in the CheckoutComplete design file as these stypes are not shared with the main gate. 

The shared css file is more for css that will be used by other gate versions when we run AB test on the main gate. I don't see us running an tests on the CheckOutComplete personlised designs yet.